README.md

```diff
@@ -8,8 +8,10 @@ MinIO uses [Sphinx](https://www.sphinx-doc.org/en/master/index.html) to generate
 
 - Any GNU/Linux Operating System.
 - python 3.10.x and python-pip
+- python3.10-venv
 - sphinx 4.3.2
 - nodejs 14.5.0 or later
+- npm
```

Comment:
I don't know which versions of npm this works with, suggestions?

Reply:
So locally at least 14.5.0 seems to be our base. That said I think we recently tested with Hugo...16.19.1 and it worked fine.

Since hugo seems to require 16.19 right now, and that's Node stable anyways, we could bump to that.